CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 1.5.0-beta.1
+
+## ✨ Features
+* GeolocateControl displays a `disabled` icon if user denies geolocation permission. [#8871](https://github.com/mapbox/mapbox-gl-js/pull/8871))
+* Add `mapboxgl.getRTLTextPluginStatus()` to query the current status of the `rtl-text-plugin` ([#8864](https://github.com/mapbox/mapbox-gl-js/pull/8864))
+* `hash` Map option can now be set as a string which sets the map hash to a custom query parameter. ([#8603](https://github.com/mapbox/mapbox-gl-js/pull/8603)) (h/t [SebCorbin](https://github.com/SebCorbin))
+
+## 🍏 Improvements
+* `icon-text-fit` now properly respects `text-writing-mode`:`vertical` ([#8835](https://github.com/mapbox/mapbox-gl-js/pull/8835))
+* GeolocateControl emits an `outofmaxbounds` and cancels the geolocate `flyTo` if the user is outside of `map.maxBounds` ([#8756](https://github.com/mapbox/mapbox-gl-js/pull/8756)) (h/t [MoradiDavijani](https://github.com/MoradiDavijani))
+* Symbols fade faster when zooming out quickly, reducing overlap. ([#8628](https://github.com/mapbox/mapbox-gl-js/pull/8628))
+* Reduce memory usage for vector tiles that contain long strings in feature properties. ( [#8863](https://github.com/mapbox/mapbox-gl-js/pull/8863))
+* Improved icon density for `text-variable-anchor` by trying additional placements when icon gets collided out. ([#8803](https://github.com/mapbox/mapbox-gl-js/pull/8803))
```

Comment:
hmm, just from reading this changelog entry, I'm a bit confused what the change was in this release because this entry seems to describe the intention of the variable label placement feature.

Reply:
The change is in the `icon` part, we wouldn't try additional placements if the `icon` associated with the text got collided out, I made the description a bit clearer.

Reply:
Got it. Reading your updated entry, I also think it's more of a bug fix, as when we release new style spec properties i.e. `text-variable-anchor`, the expectation is that they work with all existing properties i.e. `icon-text-fit`. (aside: In retrospect, maybe `symbol-variable-anchor` would have been a better name rather than `text-variable-anchor` if this property affects icons too.. 💭 )

I might rephrase as: 
> Fix `text-variable-anchor` not trying multiple placements during collision detection for icons when `icon-text-fit` is enabled.